chart/hyrax/templates/_helpers.tpl

```diff
@@ -179,6 +179,14 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "redis://:%s@%s:%s" .Values.redis.password (include "hyrax.redis.host" .) "6379/0" -}}
 {{- end -}}
 
+{{- define "hyrax.hyrax.host" -}}
```

Comment:
This template doesn't seem to be used/needed?

Reply:
you are correct - I really tied my self in knots a bit trying to keep from having to have a global. I'll remove this.